src/commands/slack.ts

```diff
@@ -97,16 +107,25 @@ class JahiaSlackReporter extends Command {
       if (failedReports.length > 0) {
         msg += '```\n'
       }
+      
       failedReports.forEach(failedReport => {
         const failedSuites = failedReport.testsuites.filter(s => s.failures > 0)
-        failedSuites.forEach(failedSuite => {
-          msg += `Suite: ${failedSuite.name} - ${failedSuite.tests.length} tests - ${failedSuite.failures} failures\n`
-          const failedTests = failedSuite.tests.filter(t => t.status ===  'FAIL')
-          failedTests.forEach(failedTest => {
-            msg += ` |-- ${failedTest.name} (${failedTest.time}s) - ${failedTest.failures.length > 1 ? failedTest.failures.length + ' failures' : ''} \n`
+        
+        // If there are more than 3 failing suites, only these 3 will be posted and then the other failures will be posted in its thread
+        if (r.failures > 3) {
+          for(var i = 0; i < 3; i++) {
+            slackMsgForSuite(msg, failedSuites[i])
+          }
+          
+          for(var j = 3; j < failedSuites.length; j++) {
+            slackMsgForSuite(threadMsg, failedSuites[j])
+          })
```

Comment:
When the result is truncated into a thread, should we also add a note such as "and xx more (see thread)" ?